build.gradle

```diff
@@ -15,7 +15,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.web3j:core:4.6.0-android'
+    implementation 'org.web3j:core:4.9.1'
```

Comment:
can't use this. Use only android flavors.
pls also check compatibility with api level 21+